application/consumer-rabbitmq-quarkus/src/test/java/webapp/tier/service/RabbitmqSubscribeServiceTest.java

```diff
@@ -1,69 +1,73 @@
 package webapp.tier.service;
 
-import static org.hamcrest.CoreMatchers.*;
-import static org.hamcrest.MatcherAssert.*;
-import static org.junit.jupiter.api.Assertions.*;
+import static org.junit.jupiter.api.Assertions.fail;
 
-import java.io.IOException;
-import java.util.concurrent.TimeoutException;
+import java.nio.charset.StandardCharsets;
+import java.util.concurrent.TimeUnit;
 
 import javax.inject.Inject;
 
 import org.junit.jupiter.api.Test;
 
+import com.github.fridujo.rabbitmq.mock.MockConnection;
 import com.github.fridujo.rabbitmq.mock.MockConnectionFactory;
+import com.rabbitmq.client.Channel;
 import com.rabbitmq.client.Connection;
 
 import io.quarkus.test.junit.QuarkusTest;
 
 @QuarkusTest
 class RabbitmqSubscribeServiceTest {
 
-	@Inject
-	RabbitmqSubscribeService svc;
+	private static final String exchangename = "exchangemsg";
+	private static final String routingkey = "routingkeymsg";
 
-	@Test
-	void testIsActiveFalse() {
-		assertThat(svc.isActive(), is(false));
+	private static MockConnection createRabbitmqMock() {
+		MockConnection conn = new MockConnectionFactory().newConnection();
+		return conn;
 	}
 
-	@Test
-	void testIsActiveTrue() throws IOException, TimeoutException {
-		RabbitmqSubscribeService rsvc = new RabbitmqSubscribeService() {
-			public Connection getConnection() {
-				Connection conn = new MockConnectionFactory().newConnection();
-				return conn;
-			}
-		};
-		assertThat(rsvc.isActive(), is(true));
-	}
+	@Inject
+	RabbitmqSubscribeService svc;
```

Comment:
Codacy found an issue: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/manual/yurake/k8s-3tier-webapp/pullRequest?prid=5997249)